### ISSUE TITLE
(Play Services Resolver 1.2.122) Is it possible to specify build settings for pods?

### ISSUE THREAD

**Issue by fmoyano** [googlesamples/unity-jar-resolver#255]
Due to [this issue](https://github.com/facebook/facebook-sdk-for-unity/issues/287), in order to compile the Facebook SDK for Unity, I would need to instruct Xcode to use GNUXX dialect instead of CXX, and I was thinking of using [pod_target_xcconfig](https://guides.cocoapods.org/syntax/podspec.html#pod_target_xcconfig). This is right now my Dependencies.xml:

```
<iosPods>
        <iosPod name="Bolts" version="~> 1.7" />
        <iosPod name="FBSDKCoreKit" version="~> 5.2"/>
        <iosPod name="FBSDKLoginKit" version="~> 5.2" />
        <iosPod name="FBSDKShareKit" version="~> 5.2" />
</iosPods>
```

How should I modify this file to add something like:

spec.pod_target_xcconfig = { 'CLANG_CXX_LANGUAGE_STANDARD' => 'gnu++11' }?

**Comment by stewartmiles** [comment]
The following code has the properties that are supported for each `pod` (see https://guides.cocoapods.org/syntax/podfile.html#pod) line in the *Dependencies.xml file.
https://github.com/googlesamples/unity-jar-resolver/blob/a00f7d0e6b8c0cb64bb6b605ac6f1fe1c62985cb/source/IOSResolver/src/IOSResolver.cs#L264

I think `spec.pod_target_xcconfig` (https://guides.cocoapods.org/syntax/podspec.html#pod_target_xcconfig) needs to be configured by the Podspec (i.e what is used to generate the Cocoapod) I'm not sure that you can do this with the Podfile.  Perhaps write a processor for the generated Xcode project instead? 
There is some code that does this in this project here https://github.com/googlesamples/unity-jar-resolver/blob/a00f7d0e6b8c0cb64bb6b605ac6f1fe1c62985cb/source/IOSResolver/src/IOSResolver.cs#L1619

**Comment by fmoyano** [comment]
Yeah, thanks, that's what I imagined but I wanted to make sure. I've actually managed to patch the Xcode project to use _gnuxx_ but I guess linker errors.

The thing is that I wanted only the files of those pods (more concretely those of FBSDK* pods) to compile with different options (_gnu++0x_ and _gnu99_ instead of _c++0x_ and _c11_) while keeping the others compile with the Xcode by default options.

I guess that it probably can be done using a post-build step to modify the Xcode project, although I don't know how to do it now. The only related thing I find in the API is proj.SetCompileFlagsForFile, but it seems overwhelming to need to specify all files and all compile flags...

Given that this is an unrelated question to the original issue I'm going to close it, although I'd be grateful if anybody could give me any hint to accomplish what I want.
